src/phpDocumentor/Reflection/Php/Visibility.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.5
+ *
+ * @copyright 2010-2015 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\Reflection\Php;
+
+
+final class Visibility
+{
+    /**
+     * @var string value can be public, protected or private
+     */
+    private $visibility;
+
+    public function __construct($visibility)
+    {
+        $visibility = strtolower($visibility);
+
+        if ($visibility !== 'public' && $visibility !== 'protected' && $visibility !== 'private') {
```

Comment:
It would be a good to extract the strings public, protected and private into class constants of this class so that other locations can re-use thos